crates/turborepo-lib/test/scripts/hello_world_hello_moon.js

```diff
@@ -0,0 +1,2 @@
+console.log("hello world");
```

Comment:
I assume this is leftover from testing?

Reply:
This script is used explicitly for testing stdout/err capture where we write `hello world` to stdout and `hello moon` to stderr.